### ISSUE TITLE
Config menu redesign

### ISSUE THREAD

**Issue by Amnesia1000** [melonDS-emu/melonDS#1536]
The current design of the configuration tab is very extensive, I could group some options to make this menu shorter, as an example:
![Menu](https://user-images.githubusercontent.com/45115804/196013296-2ffe94bb-d72a-4a82-9de3-37b7444f6495.png)
I'm not good at editing images, but there could also be "Connectivity" and within the "Multiplayer" and "WiFi" options.
All this just as an example, organize it as you think is correct, but currently the Config tab is huge.

**Comment by sandwichwater** [comment]
i feel like instead of having it branch out into yet even more sub tabs , there could be a full tab specifically for screen as that seems to be what takes up the most . i personally dont have a problem with how it is but if they were to change anything about it i feel like screen settings should all have their own tab , or maybe have it in with the video settings

**Comment by Amnesia1000** [comment]
> i feel like instead of having it branch out into yet even more sub tabs , there could be a full tab specifically for screen as that seems to be what takes up the most . i personally dont have a problem with how it is but if they were to change anything about it i feel like screen settings should all have their own tab , or maybe have it in with the video settings

It's a great idea, a window to handle all the screen options. With checkboxes or lists with the different options.

**Comment by Brankale** [comment]
Yeah, I agree with @sandwichwater. Screen seetings are the ones that I personally modify the most. Having them in a completely separate tab along File, System and Config would be great. Having them in a sub-tab instead would make this options harder to reach, and if you to experiment with them a bit it could become tiring.

**Comment by crueter** [comment]
It would make a lot more sense to make it look more like Citra's/Yuzu's configuration dialog, with individual tabs on the side and subtabs on each page; this has the benefit of allowing users to access the menus with a single shortcut, i.e. Ctrl+,
